test-utils/to-match-string.ts

```diff
@@ -1,24 +1,40 @@
 import type { RegexSequence } from '../src/types';
 import { wrapRegExp } from './utils';
 
+interface MatchTypeOptions {
+  exactString: boolean;
```

Comment:
Why is this option mandatory? It generate a lot of noise in the testing code. If possible pick one of the values `true` or `false` as default.